packages/minimal/src/index.ts

```diff
@@ -196,3 +205,11 @@ export function _callOnClient(method: string, ...args: any[]): void {
 export function startTransaction(context: TransactionContext): Transaction {
   return callOnHub('startTransaction', { ...context });
 }
+
+/**
+ * Callback that receives a Span if there is one on the Scope.
+ * @param callback Will only be invoked in case there is a Span on the Scope
+ */
+export function getSpan(callback: (span: Span) => void): void {
```

Comment:
Our APIs normally use `getFoo(...): Foo`. It is awkward that `getSpan` doesn't return a `Span` but rather calls a callback.

The similar API that works like this that comes to mind is `withScope(callback: (scope: Scope) => void): void`.